### ISSUE TITLE
Configuration properties can't be changed after construction

### ISSUE THREAD

**Issue by martinhd** [marmelab/EventDrops#24]
Please forgive me - I know very little about d3.js
But if in the example  index.html I move start and end away from the constructor to a later phase,
the HTML will not reflect these  (though the properties in the debugger seem to be correct)
(I've tested the same with other d3.js examples and there it works)

``` js
            var graph = d3.chart.eventDrops()
//move to after construction                .start(new Date(startTime))
//move to after construction                .end(new Date(endTime))
                .locale(locale)
                .eventColor(function (datum, index) {
                    return color(index);
                })
                .width(1200)
                .margin({ top: 100, left: 250, bottom: 0, right: 0 })
                .axisFormat(function(xAxis) {
                    xAxis.ticks(5);
                })
                .eventHover(function(el) { 
                    var series = el.parentNode.firstChild.innerHTML;
                    var timestamp = d3.select(el).data()[0]
                    document.getElementById('legend').innerHTML = 'Hovering [' + timestamp + '] in series "' + series + '"'; 
                });

//moved here
            graph.start = new Date(startTime);
            graph.end = new Date(endTime);
```

**Comment by ThieryMichel** [comment]
You just need to call graph(el) again to refresh the graph with the new parameter.

**Comment by martinhd** [comment]
If you modify the example EventDrops\example\custom\index.html like shown above, you will see that the new start/end times will not take effect even if graph(element) is called

**Comment by jpetitcolas** [comment]
Sorry for the delay... It should be fixed in #54.
